test/functional/treesitter/highlight_spec.lua

```diff
@@ -82,7 +82,7 @@ local hl_grid_legacy_c = [[
     {15:return} {26:0};                                                      |
   }                                                                |
   {1:~                                                                }|*2
-                                                                   |
+  {MATCH:.*}|
```

Comment:
can we check for `more lines` here or is it too unreliable?
```suggestion
  {MATCH:more lines}|
```

Reply:
Nah the 14 is consistent but this now needs to match against both an empty and `14 more lines` message area. So could either add another expected screen state, or use something like:
`{MATCH:1?4? m?o?r?e? l?i?n?e?s?.*}|`
parentheses don't seem to work here unfortunately:
`{MATCH:(14 more lines)?.*}|`.

Reply:
> parentheses don't seem to work here unfortunately

yeah, not supported by Lua patterns. `m?o?r?e? l?i?n?e?s?` seems good enough, and is almost readable :D